renderdoc/driver/vulkan/vk_stringise.cpp

```diff
@@ -4320,3 +4320,50 @@ rdcstr DoStringise(const VkGeometryTypeKHR &el)
   }
   END_ENUM_STRINGISE();
 }
+
+template <>
+rdcstr DoStringise(const VkImageCompressionFixedRateFlagBitsEXT &el)
+{
+  BEGIN_BITFIELD_STRINGISE(VkImageCompressionFixedRateFlagBitsEXT);
+  {
+    STRINGISE_BITFIELD_BIT(VK_IMAGE_COMPRESSION_FIXED_RATE_NONE_EXT)
```

Comment:
minor nit: since `NONE` is 0 this will not be stringised. 'special' values like 0 within bitfields should be handled with `STRINGISE_BITFIELD_VALUE` which does an `==` compare. Also useful for the rare case where some bits, if all set, should be handled specially - e.g. `VK_SHADER_STAGE_ALL_GRAPHICS`.